src/globals.cpp

```diff
@@ -285,7 +312,8 @@ CTweakRef<bool> enableDataSigVerifyTweak("consensus.enableDataSigVerify",
     "true if OP_DATASIGVERIFY is enabled.",
     &enableDataSigVerify);
 
-CTweak<unsigned int> numMsgHandlerThreads("net.msgHandlerThreads", "Number of message handler threads", 0);
+CTweak<unsigned int> numMsgHandlerThreads("net.msgHandlerThreads", "Max message handler threads", 0);
+CTweak<unsigned int> numTxAdmissionThreads("net.txAdmissionThreads", "Max transaction mempool admission threads", 0);
```

Comment:
what if someone try to set a too high value here? couldn't we have a validato to limit the max number of thread to same sane value?